huaweicloud/services/acceptance/apig/data_source_huaweicloud_apig_channel_members_test.go

```diff
@@ -0,0 +1,394 @@
+					resource.TestCheckResourceAttrSet(ecsAllName, "members.0.member_group_id"),
+					resource.TestCheckResourceAttrSet(ecsAllName, "members.0.member_ip_address"),
+					resource.TestCheckResourceAttrSet(ecsAllName, "members.0.ecs_id"),
+					resource.TestCheckResourceAttrSet(ecsAllName, "members.0.ecs_name"),
+					resource.TestCheckResourceAttrSet(ecsAllName, "members.0.port"),
+					resource.TestCheckResourceAttrSet(ecsAllName, "members.0.is_backup"),
+					resource.TestCheckResourceAttrSet(ecsAllName, "members.0.status"),
+					resource.TestCheckResourceAttrSet(ecsAllName, "members.0.weight"),
+					resource.TestMatchResourceAttr(ecsAllName, "members.0.create_time",
+						regexp.MustCompile(`^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}?(Z|([+-]\d{2}:\d{2}))$`)),
+					dcEcsAllName.CheckResourceExists(),
+					dcEcsFilterByName.CheckResourceExists(),
+					resource.TestCheckOutput("ecs_name_filter_is_useful", "true"),
+					dcEcsFilterByGroupName.CheckResourceExists(),
+					resource.TestCheckOutput("ecs_group_name_filter_is_useful", "true"),
+					dcIpFilterByGroupName.CheckResourceExists(),
+					resource.TestCheckOutput("ip_group_name_filter_is_useful", "true"),
+					dcEcsFilterByGroupId.CheckResourceExists(),
+					resource.TestCheckOutput("ecs_group_id_filter_is_useful", "true"),
+					dcIpFilterByGroupId.CheckResourceExists(),
+					resource.TestCheckOutput("ip_group_id_filter_is_useful", "true"),
+					dcFilterByPreciseSearch.CheckResourceExists(),
+					resource.TestCheckOutput("precise_search_filter_is_useful", "true"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDataSourceChannelMembers_Compute_base(name string) string {
+	return fmt.Sprintf(`
+data "huaweicloud_availability_zones" "test" {}
+
+data "huaweicloud_compute_flavors" "test" {
+  availability_zone = data.huaweicloud_availability_zones.test.names[0]
+  performance_type  = "normal"
+  cpu_core_count    = 2
+  memory_size       = 4
+}
+
+data "huaweicloud_vpc_subnet" "test" {
+  id = "%[2]s"
+}
+
+data "huaweicloud_images_image" "test" {
+  name        = "Ubuntu 18.04 server 64bit"
+  most_recent = true
+}
+
+data "huaweicloud_networking_secgroup" "test" {
+  name = "default"
+}
+
+resource "huaweicloud_compute_instance" "test" {
+  count = 2
+
+  name                = format("%[1]s_%%s", count.index)
+  description         = "terraform test"
+  image_id            = data.huaweicloud_images_image.test.id
+  flavor_id           = data.huaweicloud_compute_flavors.test.ids[0]
+  security_group_ids  = [data.huaweicloud_networking_secgroup.test.id]
+  stop_before_destroy = true
+  agency_name         = "test111"
+  agent_list          = "hss"
+
+  user_data = <<EOF
+#! /bin/bash
+echo user_test > /home/user.txt
+EOF
+
+  network {
+    uuid              = data.huaweicloud_vpc_subnet.test.id
+    source_dest_check = false
+  }
+
+  system_disk_type = "SAS"
+  system_disk_size = 50
+
+  data_disks {
+    type = "SAS"
+    size = "10"
+  }
+}`, name, acceptance.HW_SUBNET_ID)
```

Comment:
```suggestion
}`, name, acceptance.HW_APIG_DEDICATED_INSTANCE_USED_SUBNET_ID)
```
acceptance.TestAccPreCheckApigSubResourcesRelatedInfo(t) do not forget to check

Reply:
done